backend/webhooks/events/events.go

```diff
@@ -0,0 +1,44 @@
+package events
+
+import "github.com/teamhanko/hanko/backend/persistence/models"
+
+type Event string
+
+const (
+	User         Event = "user"
```

Comment:
This event is only used in tests and has no use anywhere else so it should be removed.